src/rsa/blinding.rs

```diff
@@ -0,0 +1,585 @@
+// Copyright 2016 David Judd.
+// Copyright 2016 Brian Smith.
+//
+// Permission to use, copy, modify, and/or distribute this software for any
+// purpose with or without fee is hereby granted, provided that the above
+// copyright notice and this permission notice appear in all copies.
+//
+// THE SOFTWARE IS PROVIDED "AS IS" AND AND THE AUTHORS DISCLAIM ALL WARRANTIES
+// WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY
+// SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+// WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+// OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+// CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+//! RSA blinding parameter generation.
+
+use {c, core, rand, error, rsa};
+use limb::*;
+
+const RSA_KEY_MAX_LIMBS: usize = rsa::RSA_PUBLIC_KEY_MODULUS_BITS_MAX / LIMB_BITS;
```

Comment:
When re-reviewing this, I realized that this should be defined next to `RSA_PUBLIC_KEY_MODULUS_BITS_MAX` and we should add a comment explaining that we assume that `RSA_PUBLIC_KEY_MODULUS_BITS_MAX` is always a multiple of `LIMB_BITS`.

Preferably, use the rounding calculation:

```
(rsa::RSA_PUBLIC_KEY_MODULUS_BITS_MAX + LIMB_BITS - 1) / LIMB_BITS.